tests/testthat/test-read_vpts.R

```diff
@@ -1,173 +1,233 @@
-vptsfile <- system.file("extdata", "example_vpts.txt", package = "bioRad")
+# Define the URLs of test files
+urls <- c(
+  "https://aloft.s3-eu-west-1.amazonaws.com/baltrad/hdf5/czbrd/2023/06/01/czbrd_vp_20230601T000000Z_0xb.h5",
+  "https://aloft.s3-eu-west-1.amazonaws.com/baltrad/hdf5/czbrd/2023/06/01/czbrd_vp_20230601T000500Z_0xb.h5",
+  "https://aloft.s3-eu-west-1.amazonaws.com/baltrad/hdf5/czbrd/2023/06/01/czbrd_vp_20230601T001000Z_0xb.h5",
+  "https://aloft.s3-eu-west-1.amazonaws.com/baltrad/monthly/bejab/2023/bejab_vpts_202303.csv.gz",
+  "https://aloft.s3-eu-west-1.amazonaws.com/baltrad/monthly/bejab/2023/bejab_vpts_202304.csv.gz",
+  "https://aloft.s3-eu-west-1.amazonaws.com/baltrad/monthly/bewid/2023/bewid_vpts_202303.csv.gz"
+)
 
-test_that("read_vpts() returns error on incorrect parameters", {
-  empty_file <- tempfile()
-  file.create(empty_file)
+# Define the path to the new temporary directory
+temp_dir <- "temp"
```

Comment:
Best practice is to use R function `tempdir()` for defining temporary directories, not creating your own temp directory in the user space

Reply:
FIxed in 62eaac0a0183db77771c2569f663c6f77ffbc15b